robocop/checkers/naming.py

```diff
@@ -770,6 +787,24 @@ def visit_LibraryImport(self, node):  # noqa
                     end_col=name_token.end_col_offset + 1,
                 )
 
+    def visit_SettingSection(self, node):  # noqa
+        self.generic_visit(node)
+        if not self.init_file:
+            return
+        for setting in node.body:
+            if not setting.data_tokens:
+                continue
+            setting_name = setting.data_tokens[0].value
+            if setting_name.lower() in ["test template", "default tags"]:
```

Comment:
If we would parse error instead, would we also support translated settings? I'm thinking about pros and cons of both implementations

Reply:
Good point. I will check it. The translations introduced a lot of complications to our tool :D

Reply:
Yes, it works for translations if this is just a properly served error. It does not work for the current implementation.
I will update the PR.

Reply:
Updated. Please review.